pkg/netflow/server.go

```diff
@@ -103,8 +104,17 @@ func (s *Server) stop() {
 }
 
 // StartServer starts the global NetFlow collector.
-func StartServer(sender aggregator.Sender) error {
-	server, err := NewNetflowServer(sender)
+func StartServer(demux aggregator.DemultiplexerWithAggregator) error {
+	epForwarder, err := demux.GetEventPlatformForwarder()
+	if err != nil {
+		return err
+	}
+
+	sender, err := demux.GetDefaultSender()
+	if err != nil {
+		return err
+	}
+	server, err := NewNetflowServer(sender, epForwarder)
 	serverInstance = server
 	return err
```

Comment:
I know that `server` would be `nil` in case of an error, but WDYT of:

```suggestion
	server, err := NewNetflowServer(sender, epForwarder)
	if err != nil {
		return err
	}
	serverInstance = server
	return nil
```

In order to make clear that `serverInstance` stays `nil` in case of an error (+ protect against a change in `NewNetflowServer` that may return something if modified.
No strong opinion, please discard my proposition if you disagree or if I miss something.